BootCampV2/Homeworks/SecondWeek/ArrayExtension.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+
+namespace BootCampV2
+{
+    public class ArrayExtension
+    {
+        /// <summary>
+        /// Use the bubble sort algorithm to sort an array
+        /// <para> return: sorted Array </para> 
+        /// </summary>
+        public static int[] BubbleSort(int[] array)
+        {
+            for (int i = 0; i < array.Length; i++)
```

Comment:
This is a good implementation of bubble sort.

* Your inner loop doesn't loop the entire array.
* You bail early if you find the rest of the array is sorted
* Nested `SwapElements` method

Good work.